### PR TITLE
feat: doppler spec

### PR DESCRIPTION
**Pull request by gusfune**
This adds specs for `doppler` as requested on issue #704

### CODE REVIEW

File: src/doppler.ts [hunk 1]
```diff
@@ -0,0 +1,875 @@
...
+  description: "Config (e.g. dev)",
+  args: { name: "string" },
+};
+
+const fallbackOption: Fig.Option = {
+  name: "f",
+  description:
+    "Path to the fallback file. encrypted secrets are written to this file after each successful fetch. secrets will be read from this file if subsequent connections are unsuccessful",
+  args: { name: "string" },
+};
```
Comment (QuiiBz) [inline src/doppler.ts:17]: Can these arguments have a [`template`](https://fig.io/docs/reference/arg#template) of `filepaths`?

File: src/doppler.ts [hunk 2]
```diff
@@ -0,0 +1,875 @@
...
+  name: "--fallback-readonly",
+  description:
+    "Disable modifying the fallback file. secrets can still be read from the file",
+};
+
+const formatOption: Fig.Option = {
+  name: "--format",
+  description:
+    'Output format. one of json, env, yaml, docker, env-no-quotes (default "json")',
+  args: { name: "string" },
```
Comment (QuiiBz) [inline src/doppler.ts:42]: Would love the see some [`suggestions`](https://fig.io/docs/reference/arg#suggestions) for this argument! (json, env, yaml...)

File: src/doppler.ts [hunk 3]
```diff
@@ -0,0 +1,875 @@
...
+  name: ["--yes", "-y"],
+  description: "Proceed without confirmation",
+};
+
+const globalOptions: Fig.Option[] = [
+  {
+    name: "--api-host",
+    description:
+      'The host address for the Doppler API (default "https://api.doppler.com")',
+    args: { name: "string", template: "filepaths" },
```
Comment (QuiiBz) [inline src/doppler.ts:101]: Should this argument have a `template` of `filepaths`? I'm not sure by looking at the description

File: src/doppler.ts [hunk 4]
```diff
@@ -0,0 +1,875 @@
...
+        helpCommand("run"),
+        noCacheOption,
+        noExitOnWriteFailureOption,
+        noFallbackOption,
+        passphraseOption,
+        projectOption,
+        {
+          name: "--command",
+          description: 'Command to execute (e.g. "echo hi")',
+          args: { name: "string" },
```
Comment (QuiiBz) [inline src/doppler.ts:640]: Can this argument have the [`isCommand`](https://fig.io/docs/reference/arg#iscommand) property?

**Review by QuiiBz — CHANGES_REQUESTED** [review]
Thanks for this PR, Doppler is such a great tool! Left some messages to resolve before merging

**Comment by gusfune** [comment]
> Thanks for this PR, Doppler is such a great tool! Left some messages to resolve before merging

Thanks a lot! I just made the adjustments.

**Review by QuiiBz — APPROVED** [review]
Thanks for making the changes, and thanks for the contribution! LGTM 🎉

**Pull request merged**